### DISCUSSION TITLE
how to connect to a cockroach db?

### DISCUSSION THREAD

**Discussion by kakarot-dev** [prisma/prisma#8711] [Q&A]
`Error: Error in connector: Error querying the database: Error querying the database: Error querying the database: db error: FATAL: codeParamsRoutingFailed: missing cluster name in connection string`

this is the error im recieveing
if anyone uses cockroach db, pls tell me how to format the connection string!

rn im using 
`DATABASE_URL="postgresql://<username>:<pass>@<cloud-id>:26257/defaultdb?sslmode=verify-full&sslcert=../.postgresql/root.crt&options=--cluster%3D<name>`

should i do it anyother way or is this correct??

**Comment by kakarot-dev** [comment]
Pls help

**Comment by ryands17** [comment]
@kakarot-dev 👋 

We do not support Cockroach DB yet. It would be great if you could add a 👍 and watch these issues for more updates :)

https://github.com/prisma/prisma/issues/2523
https://github.com/prisma/prisma/issues/4702

**Reply by kakarot-dev** [reply]
so no other way?
typeorm is too hard to configure., any eta when it can be released?

**Reply by ryands17** [reply]
We have it in our **Candidates** as per the [roadmap](https://www.notion.so/prismaio/Prisma-Roadmap-50766227b779464ab98899accb98295f) so I would say you can expect it next year and not before.

**Reply by kakarot-dev** [reply]
oooof sad, kk ill wait then

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
